src/components/molecules/TableHeader/components/EditTableTitleModal/EditTableTitleModal.tsx

```diff
@@ -1,55 +1,125 @@
+}
+
+export interface EditTableForm {
+  title: string;
+  description?: string;
+  capacity: number;
 }
 
 export const EditTableTitleModal: React.FC<EditTableTitleModalProps> = ({
   isShown,
   title,
+  description,
+  capacity,
   error,
   onSave,
   onHide,
   onCancel,
 }) => {
-  const [newTableTitle, setNewTableTitle] = useState<string>(title);
+  const {
+    register,
+    handleSubmit,
+    errors,
+    // formState,
+    // setError,
+    // clearError,
+    watch,
+  } = useForm<EditTableForm>({
+    mode: "onChange",
+    reValidateMode: "onChange",
+    defaultValues: {
+      title,
+      description,
+      capacity,
+    },
+  });
 
-  const saveChanges = useCallback(() => {
-    onSave(newTableTitle);
-    onHide();
-  }, [newTableTitle, onHide, onSave]);
+  const values = watch();
 
-  const changeTableTitle = useCallback(
-    (e: React.ChangeEvent<HTMLInputElement>) => {
-      setNewTableTitle(e.target.value);
-    },
-    []
-  );
+  const onSubmit = useCallback(() => {
+    onSave(values);
+    onHide();
+  }, [onHide, onSave, values]);
 
   return (
     <Modal show={isShown} onHide={onHide}>
       <Modal.Body>
-        <input value={newTableTitle} onChange={changeTableTitle}></input>
-        {error && <label className="input-error">{error}</label>}
-        <div className="edit-title-buttons">
-          <button className="btn btn-centered btn-secondary" onClick={onCancel}>
-            Cancel
-          </button>
-          <button
-            className="btn btn-centered btn-primary"
-            onClick={saveChanges}
-          >
-            Save Changes
-          </button>
-        </div>
+        <form onSubmit={handleSubmit(onSubmit)} className="edit-table-modal">
+          <InputField
+            ref={register({ required: true })}
+            name="title"
+            containerClassName="edit-table-modal__input--spacing"
+            placeholder="Table topic"
+          />
+          {errors.title && errors.title.type === "required" && (
+            <span className="input-error">Title is required</span>
+          )}
+          <InputField
+            ref={register}
+            name="description"
+            containerClassName="edit-table-modal__input--spacing"
+            placeholder="Describe this table (optional)"
+          />
+          <div className="edit-table-modal__capacity">
+            <label className="edit-table-modal__max-capacity">
+              Number of seats (max {MAX_TABLE_CAPACITY})
+            </label>
+            <InputField
+              ref={register({
+                required: true,
+                max: MAX_TABLE_CAPACITY,
+                min: MIN_TABLE_CAPACITY,
+              })}
+              name="capacity"
+              type="number"
+              min={MIN_TABLE_CAPACITY}
+              max={MAX_TABLE_CAPACITY}
+              placeholder="Max seats"
+            />
+          </div>
+          {errors.capacity && errors.capacity.type === "required" && (
```

Comment:
```suggestion
          {errors?.capacity.type === "required" && (
```

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1377/commits/4b7532e6f02068126c212a6f0dab1714aaaad55a